app/setup.py

```diff
@@ -29,7 +29,13 @@
 from app.publisher import LogPublisher, PubSubPublisher
 from app.secrets import SecretStore, validate_required_secrets
 from app.storage import Datastore, Dynamodb, Redis
-from app.submitter import GCSSubmitter, LogSubmitter, RabbitMQSubmitter
+from app.submitter import (
+    GCSFeedback,
+    GCSSubmitter,
+    LogFeedback,
+    LogSubmitter,
+    RabbitMQSubmitter,
```

Comment:
I think as the new feedback submitters are types of submitter they should be named something like `FeedbackGCSSubmitter` and `FeedbackLogSubmitter`.